src/main/webapp/app/shared/notification/notification-popup/notification-popup.component.html

```diff
@@ -1,13 +1,23 @@
 <div class="notification-popup-container">
     <div *ngFor="let notification of notifications; let i = index" (click)="removeNotification(i); navigateToTarget(notification)">
         <div class="icon-container d-flex flex-column justify-content-center p-4">
-            <fa-icon [icon]="'check-double'"></fa-icon>
+            <div *ngIf="notification.title == 'Live Exam Exercise Update'; else displayQuizIcon">
```

Comment:
I think this should be changed to using a constant in the typescript file instead of a hard-coded string comparison. Also why not use triple equals?

Reply:
Yes, I changed it to a constant now.

(I initially wanted to use a constant, but could not figure out how. Luckily I have found this [guide](https://www.jeffryhouser.com/index.cfm/2019/1/22/How-to-access-a-Constant-in-an-Angular-View-Template) now and it works.